cmd/kops/create_cluster.go

```diff
@@ -94,6 +97,21 @@ func (c *CreateClusterCmd) Run(args []string) error {
 		return err
 	}
 
+	if c.Manifest != "" {
+		clusterConfig, err := getManifest(c.Manifest)
+		if err != nil {
+			return err
```

Comment:
Nope, lets return the error and let something further upstream handle reporting the error.